### ISSUE TITLE
Definitely Eager to Help This Project

### ISSUE THREAD

**Issue by nebi-frame** [Song-Li/cross_browser#88]
Hi everyone,
I see that this repo is inactive even though this repo is still state-of-the-art solution as I know of even today.
I definitely want to improve this repo in coordination with @Song-Li , 
@Song-Li what are your thoughts?

**Comment by AhsanEjazC** [comment]
@nebi-frame  Leave the project as it is. Don't get it too much attention or it will stop working. You know what I mean.

**Comment by AhsanEjazC** [comment]
@nebi-frame Keep quiet and close the commit.  You know what I mean. And no, this is no longer the state of the art.

**Comment by AhsanEjazC** [comment]
@nebi-frame Keep quit.

**Comment by matiaslopezd** [comment]
Yes I agree with you @nebi-frame I'm very interested to improve this repo! @Song-Li

**Comment by nebi-frame** [comment]
But the problem is this repo is gnuv3 licensed, which makes it pretty much useless.
